mm2src/mm2_main/src/mm2.rs

```diff
@@ -356,7 +340,7 @@ pub fn mm2_main() {
     use libc::c_char;
 
     init_crash_reports();
-    log!({"AtomicDEX MarketMaker {} DT {}", MM_VERSION, MM_DATETIME});
+    println!("AtomicDEX MarketMaker {} DT {}", MM_VERSION, MM_DATETIME);
```

Comment:
If `MM_LOG` is set, would this be printed in file with old `log` macro?

Reply:
Thank you for the good note! I forgot that this line should be written to the log file.